package.json

```diff
@@ -17,5 +17,9 @@
   },
   "dependencies": {
     "rxjs": "^5.0.2"
+  },
+  "devDependencies": {
+    "react-native": "^0.43.2",
```

Comment:
Can you explain this a bit more? For me a normal development process would mean to use `npm link` to link this library in another playground project. This would be the place to include react-native IMHO. Adding it here introduces download costs to people wanting to contribute and clutters our devDependencies